finetune.py

```diff
@@ -0,0 +1,184 @@
+"""Finetuning loop."""
+
+import os
+import logging
+import argparse
+import time
+
+import torch
+from orb_models.forcefield import pretrained
+from orb_models.finetune_utilities import experiment, optim
+from orb_models.dataset import data_loaders
+from orb_models.finetune_utilities import steps
+
+
+logging.basicConfig(
+    level=logging.INFO, format="%(asctime)s - %(levelname)s - %(message)s"
+)
+
+
+def run(args):
+    """Training Loop.
+
+    Args:
+        config (DictConfig): Config for training loop.
+    """
+    device = experiment.init_device()
+    experiment.seed_everything(args.random_seed)
+
+    # Make sure to use this flag for matmuls on A100 and H100 GPUs.
+    torch.set_float32_matmul_precision("high")
+
+    # Instantiate model
+    model = pretrained.orb_v1(device=device)
+    for param in model.parameters():
+        param.requires_grad = True
+    model_params = sum(p.numel() for p in model.parameters() if p.requires_grad)
+    logging.info(f"Model has {model_params} trainable parameters.")
+
+    # Move model to correct device.
+    model.to(device=device)
+    total_steps = args.max_epochs * args.num_steps
+    optimizer, lr_scheduler = optim.get_optim(args.lr, total_steps, model)
+
+    wandb_run = None
+    # Logger instantiation/configuration
+    if args.wandb:
+        import wandb
+
+        logging.info("Instantiating WandbLogger.")
+        wandb_run = experiment.init_wandb_from_config(args, job_type="finetuning")
+
+        wandb.define_metric("global_step")
+        wandb.define_metric("epochs")
+        wandb.define_metric("train_step/*", step_metric="global_step")
+        wandb.define_metric("learning_rates/*", step_metric="global_step")
+        wandb.define_metric("finetune/*", step_metric="epochs")
+        wandb.define_metric("key-metrics/*", step_metric="epochs")
+
+    loader_args = dict(
+        dataset=args.dataset,
+        path=args.data_path,
+        num_workers=args.num_workers,
+        batch_size=args.batch_size,
+        target_config={"graph": ["energy", "stress"], "node": ["forces"]},
+    )
+    train_loader = data_loaders.build_train_loader(
+        **loader_args,
+        augmentation=getattr(args, "augmentation", True),
```

Comment:
`augmentation` is not an arg option, and either way it should be fixed to true in this example script

Reply:
Set to True and default true in `build_train_loader` as well.